src/main/java/org/example/skb/entity/Builder.java

```diff
@@ -0,0 +1,17 @@
+package org.example.skb.entity;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Builder implements Worker{
+
+    @Override
+    public void works(){
+        System.out.println("The builder went to build a house");

```

Comment:
It's better to use one logging option.  I would replace it with Log4j2, given that it is used elsewhere.